src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ChatUi.module.scss

```
@@ -277,6 +285,11 @@
   .footer {
     display: none;
   }
+  // Footer is hidden, so the chat area must not keep reserving space for it.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)